common/export.py

```diff
@@ -370,7 +376,10 @@ def csv_export_impact_comparison(compared_impacts, folder):
             rows.append(row)
 
     df = pd.DataFrame(rows)
-    df.to_csv(f"{PROJECT_ROOT_DIR}/data/{folder}/{COMPARED_IMPACTS_FILE}", index=False)
+    df.to_csv(
+        f"{PROJECT_ROOT_DIR}/{folder}/{settings.compared_impacts_file}",
```

Comment:
We should now use `os.path.join(PROJECT_ROOT_DIR, …)` everywhere as you already updated that earlier